config/release/kubemacpool.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   labels:
     control-plane: mac-controller-manager
     controller-tools.k8s.io: "1.0"
-    kubemacpool/ignoreAdmission: "true"
+    runlevel: "0"
```

Comment:
Would it make sense to set the OpenShift `runlevel` too? In case it is being read by someone.

Reply:
done